src/elements/common/selectors/version.js

```diff
@@ -34,11 +34,12 @@ const getVersionAction = ({ restored_at, trashed_at, version_promoted }: $Shape<
 
 const getVersionUser = ({
     modified_by,
+    promoted_by,
     restored_by,
     trashed_by,
     uploader_display_name,
 }: $Shape<BoxItemVersion>): User => {
-    const { name, id, ...rest } = restored_by || trashed_by || modified_by || PLACEHOLDER_USER;
+    const { name, id, ...rest } = restored_by || trashed_by || promoted_by || modified_by || PLACEHOLDER_USER;
```

Comment:
I think we should update the test for this selector to include `promoted_by`.  Do we need any other test to ensure that `promoted_by` is being used on the sidebar?

Reply:
Update the test for this code to include the promoted_by. I don't see anywhere else that needs updated

Reply:
I was thinking having a test at this level too https://github.com/box/box-ui-elements/blob/f950772922fd75089360de76b3ed39ecb035f678/src/elements/content-sidebar/versions/VersionsItem.js#L141-L146 because thats where its changing the sentence that will display the name you want to present

Reply:
assuming this is the component that is shown in your image in your description

Reply:
Included this file as well